.github/pull_request_template.md

```diff
@@ -11,7 +11,6 @@ _Optional: add details about technical approach, solutions etc._
 
 _Optional: reference dependencies to other pull requests etc._
 
-
```

Comment:
I've been deleting this line for every PR, so I decided to just do it once and for all here.
Please let me know if you prefer this to be reverted or done in another PR.